packages/driver/src/multi-domain/communicator.ts

```diff
@@ -8,11 +8,11 @@ const debug = debugFn('cypress:driver:multi-domain')
 
 const CROSS_DOMAIN_PREFIX = 'cross:domain:'
 
-const preprocessErrorForPostMessage = (value) => {
+const serializeErrorForPostMessage = (value) => {
```

Comment:
This function isn't actually serializing, the actual serializing is done by the postMessage. This function massages the object into a shape that can be serialized.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/20247/commits/d40791761dc9018b5383a2ee6ed6862c80ddb82c